zkstack_cli/crates/zkstack/src/commands/chain/init/mod.rs

```diff
@@ -58,11 +58,15 @@ async fn run_init(args: InitArgs, shell: &Shell) -> anyhow::Result<()> {
     let chain_config = config
         .load_current_chain()
         .context(MSG_CHAIN_NOT_FOUND_ERR)?;
+
+    if args.update_submodules.is_none() || args.update_submodules == Some(true) {
```

Comment:
this change is needed to be able to switch between different contracts commits, since when submodules are updated, they will be changed back to the current ones